fosdem-bare-metal-in-the-cloud/source/index.rst

```diff
@@ -0,0 +1,65 @@
+
+
+Bare Metal In The Cloud: Isn’t it Ironic?
+=========================================
+
+*by Dmitry Tantsur and Ilya Etingof, Red Hat*
+
+In this talk
+------------
+
+* Why allocating bare metal machines
+* Ironic introduction, architecture and workflows
+* Future features
+
+Why bare metal allocation
+-------------------------
+
+* Raw computing power
+* Hard-to-virtualize hardware e.g. GPU
+* Perfect isolation e.g. tenant security
+
+What's ironic
+-------------
+
+* OpenStack bare metal provisioning service
+* + OpenStack Nova driver
+* + Deployment Agent
+
+What's inside ironic
```

Comment:
Since we only have 20 minutes, I'd skip this and concentrate on user-visible features.

Reply:
Agree